app/src/common/shared/org/mozilla/vrbrowser/browser/engine/Session.java

```diff
@@ -1239,6 +1231,25 @@ public void onContentBlocked(@NonNull final GeckoSession session, @NonNull final
         }
     }
 
+    @Override
+    public void onContentLoaded(@NonNull GeckoSession geckoSession, @NonNull ContentBlocking.BlockEvent event) {
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.AD) != 0) {
+            Log.i(LOGTAG, "Loading Ad: " + event.uri);
+        }
+
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.ANALYTIC) != 0) {
+            Log.i(LOGTAG, "Loading Analytic: " + event.uri);
+        }
+
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.CONTENT) != 0) {
+            Log.i(LOGTAG, "Loading Content: " + event.uri);
+        }
+
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.SOCIAL) != 0) {
+            Log.i(LOGTAG, "Loading Social: " + event.uri);
+        }
+    }
```

Comment:
These need to be `Log.d` so they get stripped out of release builds or we will get rejected from HTC store.